moped-database/migrations/1696622988854_add_feature_length_column/up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE feature_street_segments
+    ADD COLUMN length_feet INTEGER GENERATED always AS (
+ST_Length(geography) * 3.28084) stored;
```

Comment:
Meters, of course! 3.28084 feet in a meter. I was confused for a second until I ran down that `ST_Length(geography)` is always in meters. 👍